### DISCUSSION TITLE
Frigate and camera MQTT events

### DISCUSSION THREAD

**Discussion by JanSuklje** [blakeblackshear/frigate#6682] [Ask A Question]
I have a camera that has person detection backed in and since I can't get my hands on Google Coral, I was wondering if its possible to utilize the camera, so when it detects a person, Frigate registers that as an event.

So in Frigate I would have detection disabled and as I understand it, its possible to listen to camera events such as movement and such using MQTT. Im really new to all of this and would appreciate any help with this as im not even sure where to start.

Any idea on how I can find out what MQTT data the camera is sending (if any) and how I can subscribe to it?
If I have that figured out, how can I tell frigate to mark that as an event/detection so it shows it like it shows any detection using backed in detectors?

**Comment by NickM-27** [comment]
This could technically be done using frigate 0.13 (currently in dev) and HomeAssistant but I don't think first party support for such use cases will happen
